### PR TITLE
bump phpstan dependencies for ^1

### LINKED ISSUE #37: Support phpstan 1
Hi. I request update to phpstan:^1.

It would be need to change like 
https://github.com/rectorphp/rector/issues/6744

### PR DESCRIPTION
**Pull request by sasezaki**
fix #37

### CODE REVIEW

**Comment by sasezaki** [comment]
Later , I will fix these.

```
 ------ ------------------------------------------------------------------------------------------------
  Line   lib/NotNow/NoRelativeDateTimeInterfaceRule.php
 ------ ------------------------------------------------------------------------------------------------
  48     Access to an undefined property PhpParser\Node\Arg|PhpParser\Node\VariadicPlaceholder::$value.
         💡 Use ->getArgs() instead of ->args.
 ------ ------------------------------------------------------------------------------------------------

 ------ ------------------------------------------------------------------------------------------------
  Line   lib/NotNow/NoRelativeStrtotimeRule.php
 ------ ------------------------------------------------------------------------------------------------
  48     Access to an undefined property PhpParser\Node\Arg|PhpParser\Node\VariadicPlaceholder::$value.
         💡 Use ->getArgs() instead of ->args.
 ------ ------------------------------------------------------------------------------------------------


 [ERROR] Found 2 errors

```

**Comment by Slamdunk** [comment]
Thank you for taking care of this :+1:

File: phpstan.neon [hunk 1]
```diff
@@ -11,10 +11,11 @@ parameters:
     paths:
         - lib/
         - tests/
-    excludes_analyse:
-        - tests/NotNow/TestAsset/
-        - tests/TestAsset/
-        - tests/TmpAsset/
+    excludePaths:
```
Comment (sasezaki) [inline phpstan.neon:14]: phpstan said.

```
Parameter excludes_analyse has been deprecated so use excludePaths only from now on.
```

**Review by Slamdunk — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you @sasezaki

**Pull request merged**